core/src/main/scala/org/apache/spark/deploy/SparkHadoopUtil.scala

```diff
@@ -426,6 +426,10 @@ class SparkHadoopUtil extends Logging {
     ugi.getAuthenticationMethod() == UserGroupInformation.AuthenticationMethod.PROXY
   }
 
+  def getTimeZone: TimeZone = {
+    TimeZone.getTimeZone(sparkConf.get("spark.history.timeZone", "GMT"))
```

Comment:
It's a bit confusing using `spark.history.timeZone` property in getTimeZone method of SparkHadoopUtil.